src/js/stores/MesosSummaryStore.js

```diff
@@ -38,12 +37,7 @@ function startPolling() {
       } else {
         // If not active, push null placeholder. This will ensure we maintain
         // history when navigating back, for case where history server is down.
-
-        // Use {silent: true} Because we only want to push a summary on the stack without side
-        // effects (like re-rendering etc). The tab is out of focus so we
-        // don't want it to do any work. It only matters that there is
-        // appropriate history when we return focus to the tab.
-        MesosSummaryStore.processSummaryError({silent: true});
+        MesosSummaryActions.fetchSummary(undefined, true);
```

Comment:
I'll change it back to the `processSummaryError({silent: true});` method call as the refactoring is actually not in the scope of this PR. But I'd like to discuss this in our next meeting, as I think that actions, following the flux pattern, are the right place to do that kind of stuff.

/cc @mlunoe